### ISSUE TITLE
Add reply support for auto-censor

### ISSUE THREAD

**Issue by cbrxyz** [cbrxyz/pi-bot#318]
Auto-censor should show message that was replied to

**Comment by Nydauron** [comment]
We'll prob have to wait for the API to allow replies to be applied for webhooks.

**Comment by cbrxyz** [comment]
For full reply support, yes, but I just thought for now the webhook would prepend the original censored message with something like:
*Replied to user: Message text is here*

**Comment by Nydauron** [comment]
It would also require Message objects to contain a `reply` attribute to distinguish if the message is a reply in the first place. As of `1.6.0`, the API only offers `Message.reply()` which replies to the message object directly. I don't even know if reply data can even be fetched currently. Probably best to hold off on this til Discord decides to update with these features.

**Comment by cbrxyz** [comment]
Yep, looks like you are right on that one. Thanks for looking into that!

**Comment by pandabear189** [comment]
> It would also require Message objects to contain a `reply` attribute to distinguish if the message is a reply in the first place. As of `1.6.0`, the API only offers `Message.reply()` which replies to the message object directly. I don't even know if reply data can even be fetched currently. Probably best to hold off on this til Discord decides to update with these features.

Using, `Message.reference` is an option: https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.reference

**Comment by pandabear189** [comment]
> Looking at the 2.0 docs now, it looks like they finally made `WebsocketMessage` inherit `discord.Message` finally, so `reply()` is implemented. We can just use that.
> 
> https://discordpy.readthedocs.io/en/latest/api.html#discord.WebhookMessage.reply

Isn't webhook message the webhook you sent? I don't think webhooks can `reply` to other messages yet

**Comment by cbrxyz** [comment]
It looks like Eric is correct. Doesn’t appear to be implemented yet in the Discord API.
